cernopendata/config.py

```diff
@@ -57,6 +57,12 @@
         "LOGGING_SENTRY_CELERY", False
     )
 
+# Security
+# ========
+#: Flask-Talisman secure headers
+APP_ENABLE_SECURE_HEADERS = False
```

Comment:
Disabling HTTPS temporarily. Flask-Talisman would force HTTPS on the clients but we don't have SSL setup in docker-compose production, so installation would break now.